forge-gui/res/cardsfolder/upcoming/arvad_weatherlight_smuggler.txt

```diff
@@ -0,0 +1,16 @@
+Name:Arvad, Weatherlight Smuggler
+ManaCost:W B
+Types:Legendary Creature Vampire Knight
+PT:1/1
+K:Deathtouch
+K:Lifelink
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | CheckSVar$ X | Execute$ DBStore | TriggerDescription$ At the beginning of your end step, if a creature died this turn, CARDNAME perpetually gets +X/+X where X is the number of creatures that died this turn. This ability also triggers if NICKNAME is in your graveyard.
```

Comment:
A single trigger with both TriggerZones should work

Reply:
Use a comma?